x-pack/plugins/ingest_manager/server/services/epm/packages/get_objects.ts

```diff
@@ -59,10 +59,10 @@ export async function getObject(key: string) {
   // convert that to an object
   const asset: ArchiveAsset = JSON.parse(json);
 
-  const { type, file } = Registry.pathParts(key);
+  const { type } = Registry.pathParts(key);
   const savedObject: SavedObjectToBe = {
     type,
-    id: file.replace('.json', ''),
+    id: asset.id,
```

Comment:
IIRC, EPM initially stored the ID given by the registry/package. However, IIRC, it was removed because we couldn't count on it(upgrades?). I believe we were going to have EPM maintain a mapping(?).

/cc @ruflin @skh @neptunian 

I guess we can add this now and fix the broken case you show, then deal with the other issues if/when they show up.

Reply:
My long term preference would be that we use our "own" ids to make sure we don't conflict with other assets and keep a list.

+1 on adding this now to make it work and figure out the exact renaming. 

@skh @neptunian Your input on where we are standing here would be great.

Reply:
@jfsiii I think I can actually revert this change based on a recommendation from @ruflin. The dashboard I was exporting was relying on a custom index pattern I created instead of the one that the ingest manager creates so I should be able to get around needing to change this line.